scripts/utils.js

```diff
@@ -49,6 +49,20 @@ function getWaybackUrlFromResponse(response) {
   }
 }
 
+/**
+ * Customizes error handling
+ * @param status {string}
+ * @return {string}
+ */
+function getErrorMessage(status){
+  if(status === "timeout"){
+    return "The service you have requested has timed out."
```

Comment:
Which service is that? When I see this msg in the browser console, I don't know which of the various extensions I have displays this message. Maybe say: "The archive.org service you have requested"